src/services/models/Schema.ts

```diff
@@ -355,6 +361,44 @@ export class SchemaModel {
       return innerSchema;
     });
   }
+
+  private initConditionalOperators(schema: OpenAPISchema, parser: OpenAPIParser) {
+    const {
+      if: ifOperator,
+      else: elseOperator = {},
+      then: thenOperator = {},
+      ...clearSchema
+    } = schema;
+    const ifOperatorTitle =
+      (ifOperator && ifOperator['x-displayName']) || ifOperator?.title || 'if';
+    const elseOperatorTitle =
+      (elseOperator && elseOperator['x-displayName']) || elseOperator?.title || 'else';
+    const groupedOperators = [
+      mergeObjects(
+        {},
+        clearSchema,
+        { allOf: [ifOperator, thenOperator] },
+        { title: ifOperatorTitle },
+      ),
+      mergeObjects({}, clearSchema, elseOperator, { title: elseOperatorTitle }),
+    ];
+    this.conditionalEnum = [ifOperatorTitle, elseOperatorTitle];
+
+    this.oneOf = groupedOperators.map((variant, idx) => {
+      const merged = parser.mergeAllOf(parser.deref(variant || {}), this.pointer + '/oneOf/' + idx);
```

Comment:
check if `parser.mergeAllOf` is needed, I think SchemaModel already merged allOf.